resolver.go

```diff
@@ -36,12 +36,18 @@ const (
 	ptrIPv4domain = ".in-addr.arpa."
 	ptrIPv6domain = ".ip6.arpa."
 	respTTL       = 1800
+	maxExtDNS     = 3 //max number of external servers to try
 )
 
+type extDNSEntry struct {
+	IP   string
```

Comment:
Also, please do not export fields in an non-exported structure (have them start with lower case).